stdlib/fileinput.pyi

```diff
@@ -2,7 +2,11 @@ import sys
 from _typeshed import Self, StrOrBytesPath
 from collections.abc import Callable, Iterable, Iterator
 from types import TracebackType
-from typing import IO, Any, AnyStr, Generic
+from typing import IO, Any, AnyStr, Generic, Protocol, TypeVar, overload
+from typing_extensions import Literal
```

Comment:
```suggestion
from typing_extensions import Literal, TypeAlias
```

Reply:
Thanks, I keep forgetting! You do know you can also edit the file directly in https://github.com/JelleZijlstra/typeshed/edit/fileinput/stdlib/fileinput.pyi?pr=%2Fpython%2Ftypeshed%2Fpull%2F7934, right?

Reply:
I do know, yes :) but thanks!